lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -319,6 +319,12 @@ Ufe::UndoableCommand::Ptr UsdAttributes::removeAttributeCmd(const std::string& n
 {
     return UsdRemoveAttributeCommand::create(fItem, name);
 }
+
+Ufe::UndoableCommand::Ptr
```

Comment:
As Pierre mentioned somewhere else, this code is new to UFE 0.4.33; it isn't available in 0.4.24 where add/remove was added. You will have to move this new code into separate block that will be guarded with `#if (UFE_PREVIEW_VERSION_NUM >= 4033)` (see line 311).